app/containers/TripDetailPage/selectors.js

```diff
@@ -34,6 +34,14 @@ const selectOfferId = () => createSelector(
   (offerState) => offerState.toJS().tripId
 );
 
+const selectTripForOfferDetail = () => createSelector(
+  selectTripDetailPageDomain,
+  (offerPageState) => {
+    console.log(offerPageState.toJS())
```

Comment:
remove this

Reply:
tried deleting and it broke--did you mean something specific in this?

Reply:
Yeah just the console.log